include/geometry/triangle.h

```diff
@@ -0,0 +1,29 @@
+#ifndef GEOMETRY_TRIANGLE_H
+#define GEOMETRY_TRIANGLE_H
+
+#include <cstddef>
+#include <geometry/point.h>
+#include <geometry/vetor.h>
+#include <assert.h> 
+
+struct Triangle {
+    Point vertices[3]; 
+
+    Triangle();
+    Triangle(const Point& v1, const Point& v2, const Point& v3);
+
+    double area() const;
+    Point centroid() const;
+    Vetor normal() const;
+    bool contains(const Point& p) const; 
+
+    const Point& getVertex(int index) const {
```

Comment:
```suggestion
    Point getVertex(int index) const {
```
Quando for do tipo `struct`, a gente quer retornar a cópia mesmo.